server/test/services/migration/ProgramMigrationServiceTest.java

```diff
@@ -638,23 +646,31 @@ public void updateEnumeratorIdsAndSaveAllQuestions_noEnumeratorIds_savesProgramA
 
   @Test
   public void
-      updateEnumeratorIdsAndSaveAllQuestions_withEnumeratorIds_updatesProgramQuestionsBeforeSaving()
+      updateEnumeratorIdsAndSaveQuestions_withEnumeratorIds_updatesProgramQuestionsBeforeSaving()
           throws Exception {
     // Create child questions with enumerator IDs
     QuestionDefinition childQuestion1 =
         createTextQuestionWithEnumerator(QUESTION_2_NAME, 2L, Optional.of(QUESTION_1.getId()));
     QuestionDefinition childQuestion2 =
         createTextQuestionWithEnumerator(QUESTION_3_NAME, 3L, Optional.of(QUESTION_1.getId()));
-    ImmutableList<QuestionDefinition> questionDefinitions =
+    QuestionDefinition childQuestion3 =
+        createTextQuestionWithEnumerator("another-question", 4L, Optional.of(QUESTION_1.getId()));
```

Comment:
nitpick (non-blocking): create a local var `QUESTION_4_NAME` to store "another-question" in so you can reference it in multiple places.